ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -182,7 +182,7 @@ presubmits:
           secretName: sentry-dsn
     trigger: '(?m)^/test (?:.*? )?e2e-aws-upgrade(?: .*?)?$'
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
I think we want a longer track-record of success before we do this (although it's really up to the MCO team).  Currently, [the past 24 hours have three failures and no success][1] for this job.

[1]: https://ci-search-ci-search-next.svc.ci.openshift.org/chart?name=e2e-restore-cluster-state

Reply:
We can't have passing tests before MCO scripts are debugged - and we can't properly test those without having a dedicated test for DR scenarios (e.g. https://github.com/openshift/machine-config-operator/pull/793#discussion_r287329087)

Reply:
You can `/test e2e-restore-cluster-state` in that PR and it will run (and rerun after each bump) to help you debug that PR.  No need to run this in all other MCO PRs while tou debug that one.

Reply:
Its not clear which MCO PR would break DR scenarios. Also, if this test is misbehaving it can be skipped with `/skip` since its optional